src/pages/home/report/ReportActionsList.js

```diff
@@ -112,21 +119,21 @@ function ReportActionsList({
     personalDetailsList,
     currentUserPersonalDetails,
     hasOutstandingIOU,
-    loadMoreChats,
+    loadNewerChats,
+    loadOlderChats,
     onLayout,
     isComposerFullSize,
 }) {
     const reportScrollManager = useReportScrollManager();
     const {translate} = useLocalize();
-    const {isOffline} = useNetwork();
     const route = useRoute();
     const opacity = useSharedValue(0);
     const userActiveSince = useRef(null);
     const [currentUnreadMarker, setCurrentUnreadMarker] = useState(null);
     const scrollingVerticalOffset = useRef(0);
     const readActionSkipped = useRef(false);
+    const firstComponentsRenderRef = useRef({header: true, footer: true});
```

Comment:
I think it would be clearer to split this up into two refs and give them a different name:

```js
const hasHeaderRendered = useRef(false);
const hasFooterRendered = useRef(false);

...
...

const listFooterComponent = useCallback(() => {
    // Skip this hook on the first render, as we are not sure if more actions are going to be loaded
    // Therefore showing the skeleton on footer might be misleading
    if (!hasFooterRendered.current) {
        hasFooterRendered.current = true;
        return null;
    }

    return (
        <ListBoundaryLoader
            type={CONST.LIST_COMPONENTS.FOOTER}
            isLoadingOlderReportActions={isLoadingOlderReportActions}
            isLoadingInitialReportActions={isLoadingInitialReportActions}
            lastReportActionName={lastReportAction.actionName}
        />
    );
}, [isLoadingInitialReportActions, isLoadingOlderReportActions, lastReportAction.actionName]);
```

Reply:
Okay, will split it 👍🏻